funcs/func_cut.c

```diff
@@ -75,6 +75,11 @@
 		</syntax>
 		<description>
 			<para>Cut out information from a string (<replaceable>varname</replaceable>), based upon a named delimiter.</para>
+					<example title="The 'varname' parameter could only accept a variable name, not a variable expression">
+					<para>Beleow are examples illustrating the correct and invalid syntax.</para>
```

Comment:
Below is misspelled, and <para> tags do not belong inside an example, this should go before the example tag.

Reply:
fixed thank you